### ISSUE TITLE
[core] BinaryXMLParser injects xmlns: from other files on load

### ISSUE THREAD

**Issue by S-trace** [skylot/jadx#796]
**Checks before report**
- check [latest unstable build](https://bintray.com/skylot/jadx/unstable/_latestVersion#files) (maybe issue already fixed)
Not fixed
- check [Troubleshooting Q&A](https://github.com/skylot/jadx/wiki/Troubleshooting-Q&A) section on wiki
Not out-of-memory problem.
- search existing issues by exception message
No exception message.

**Describe error**
- provide full name of method or class with error
Not method.
- provide full java stacktrace
No stacktrace.
  **Note**: no need to copy method fallback code (commented pseudocode)
- attach or provide link to apk file (double check apk version)
[res.apk.zip](https://github.com/skylot/jadx/files/3957716/res.apk.zip)

  **Note**: GitHub don't allow attach files with `.apk` extension, but you can change extension by adding `.zip` at the end :)

Hello.

jadx-gui injects xmlns: attribute from another XML files to newly opened files, even if this xmlns: is not present in the opened file.

To check: 
1. Open attached apk, open abc_fade_in.xml, copy its content to compare tool
2. Close jadx-gui
2. Open attached apk, open avd_show_password.xml then open abc_fade_in.xml, copy its content to compare tool
3. Compare abc_fade_in.xml content from both times, take a note about xmlns:aapt appeared

I pushed a fix to https://github.com/S-trace/jadx/tree/nsmap_reset
@skylot Could you please review it and allow me to open a pull request?

Thank you.

**Comment by skylot** [comment]
@S-trace what is a good catch, thanks! Please open PR.

P.S. Next time if you already made a fix for some new issue just open PR. By line from contributiong guide:
> Before open a PR please discuss the change you wish to make via issue. PR without corresponding issue will be rejected.

I mean it is better to discuss change **before starting working** on this change especially if it is big.
So if you already implement the fix you can just open PR without asking my permission :)
